### DISCUSSION TITLE
gist github mobile

### DISCUSSION THREAD

**Discussion by snakejke** [community/community#4862] [Mobile]
Just show what I see. Google Chrome, 7inch display
![Screenshot_2021-08-01-01-01-49-462_com android chrome](https://user-images.githubusercontent.com/43587877/127753510-5e367748-9ae4-4ef0-a61b-53b9ffea6275.jpg)
I think it shouldn't be that way. I reloaded the browser and the result was the same.

**Comment by shapkarin** [comment]
First time I saw it around 4 years ago. I found that current bug is related to the gist filename. You can see it when there is a one long word for a gist name, for example Github's generated names `gist:9f973d2866fdd083ef30fe4fcc955fbb`.

There is also another markup bug.

### The steps to reproduce bug from :
1. Go to https://gist.github.com
2. Click on hamburger icon at the top right 
<img width="371" alt="Screenshot 2021-08-01 at 06 06 53" src="https://user-images.githubusercontent.com/1463086/127757820-94f79e80-2a66-40a8-9928-121064e45838.png">

3. Fill the search input and perform the search. You can write `9f973d2866fdd083ef30fe4fcc955fbb` to find the gist I created to show the issue.
4. Page will looks looks broken

<img width="390" alt="Screenshot 2021-08-01 at 06 26 14" src="https://user-images.githubusercontent.com/1463086/127758152-83cae3df-58ea-4842-9150-f2d3597fc1cb.png">

As you can see at step 4 there is also also a problem with language filter label.

<img width="390" alt="Screenshot 2021-08-01 at 06 26 14" src="https://user-images.githubusercontent.com/1463086/127758516-0e218d1c-44c7-44e7-a4ad-f2424d79b80c.png">
